hc/api/tests/test_create_check.py

```diff
@@ -77,5 +83,22 @@ def test_it_rejects_non_string_name(self):
         self.post({"api_key": "abc", "name": False},
                   expected_error="name is not a string")
 
-    ### Test for the assignment of channels
-    ### Test for the 'timeout is too small' and 'timeout is too large' errors
+    # ### Test for the assignment of channels
+    def test_it_assigns_channels(self):
+        channel = Channel(user=self.alice)
+        channel.save()
+
+        r = self.post({"api_key": "abc", "channels": "*"})
+
+        self.assertEqual(r.status_code, 201)
+        check = Check.objects.get()
+        self.assertEqual(check.channel_set.get(), channel)
+
+    # ### Test for the 'timeout is too small' and 'timeout is too large' errors
+    def test_it_rejects_small_timeout(self):
+        self.post({"api_key": "abc", "timeout": 0},
+                  expected_error="timeout is too small")
+
+    def test_it_rejects_large_timeout(self):
```

Comment:
I don't think this test is complete either

Reply:
Yes, it is incomplete. I have fixed it. Thanks for pointing that out.